src/index.js

```diff
@@ -68,7 +68,9 @@ class TCP {
       multiaddrs = [multiaddrs]
     }
     return multiaddrs.filter((ma) => {
-      if (includes(ma.protoNames(), 'ipfs')) {
+      if (includes(ma.protoNames(), 'p2p-circuit')) {
+        return false
+      } else if (includes(ma.protoNames(), 'ipfs')) {
```

Comment:
I do, but decapsulated will strip the `ipfs` part along with `p2p-circuit`, so mafmt wont match it. Mafmt does have all the logic to do the matches correctly tho.